test/envtest/suite_test.go

```diff
@@ -141,6 +141,14 @@ func down() {
 	ctx = context.TODO()
 
 	cancel()
+
+	ds := &apps.DaemonSet{}
+
+	for _, name := range []string{"dsa", "dlb", "fpga", "gpu", "iaa", "qat", "sgx"} {
```

Comment:
let's have each device do their own cleanup and clean the CR too (or rather instead)